### PR TITLE
update to business case overview and industry template

### PR DESCRIPTION
**Pull request by onthebreeze**
Pulled half of the material from google drive to UNTP site on business case
 - Significant update to the business case overview page.
 - Added more benefits and evidence to industry business case template
 - created a separate page for government business case

Still to do

- content on cost side of industry case
- government business case page
- update to community activation page.

### CODE REVIEW

**Review by zachzeus — APPROVED** [review]
Good next step

**Comment by zachzeus** [comment]
@onthebreeze  - I'm not sure what to do with the yarn.lock conflicting file.

**Review by nissimsan — APPROVED** [review]
This is great!

File: website/docs/business-case/BusinessCaseIndustry.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+* *References* Asian Development Bank (ADB), [Trade Finance Gaps Growth and Jobs Survey 2021](https://www.adb.org/publications/2021-trade-finance-gaps-growth-jobs-survey), 
+ [Trade finance gaps growth jobs survey 2023](https://www.adb.org/publications/2023-trade-finance-gaps-growth-jobs-survey). [Deep-Tier Supply Chain Finance 2022](https://www.adb.org/publications/deep-tier-supply-chain-finance)
+
+#### Reduced Finance Costs
+
+According to the International Finance Corporation (IFC), companies that adopt sustainable practices can reduce their financing costs by up to 20% due to lower risk premiums and better access to capital.
+
+* *References* International Finance Corporation (IFC), "Sustainable Finance: Creating Value for Companies and Investors," 2020.
```
Comment (LucaBrunello1) [inline website/docs/business-case/BusinessCaseIndustry.md:77]: I've scanned the internet, and even checked every 2020 publication of IFC, but cannot find this publication..

File: website/docs/business-case/BusinessCaseIndustry.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+#### Improved margins
+
+A study by the Global Reporting Initiative (GRI) found that companies with strong ESG performance can achieve up to a 10% improvement in profit margins due to enhanced operational efficiencies and lower financing costs.
+
+* *References* Global Reporting Initiative (GRI), "The Business Case for ESG: How Sustainability Can Drive Financial Performance," 2019.
```
Comment (LucaBrunello1) [inline website/docs/business-case/BusinessCaseIndustry.md:83]: Same as with IFC, nothing to be found on this one

File: website/docs/business-case/BusinessCaseIndustry.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+#### Cost of Goods Sold
+A report by McKinsey & Company indicates that companies with optimized supply chain financing can reduce their cost of goods sold by 5% to 10% due to lower financing costs and improved supply chain efficiencies.
+
+* *References* McKinsey & Company, "Supply Chain Finance: The Key to Unlocking Working Capital," 2019.
```
Comment (LucaBrunello1) [inline website/docs/business-case/BusinessCaseIndustry.md:88]: Also for this one nothing to be found on the internet or Mckinsey website. Perhaps who has put these references, has access to some private databases, PDFs or something like that? Could you put where these are coming from?

File: website/docs/business-case/BusinessCaseIndustry.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+Companies can expect a 10% to 20% reduction in operational costs due to automation and improved data accuracy. Improved supply chain visibility can reduce inventory holding costs by 15% to 30% and decrease stock-outs by 20%. Data-driven decision-making can increase productivity by 5% to 10% and enhance profitability by 3% to 5%. Enhanced customer satisfaction can lead to a 10% increase in repeat business and a 5% boost in overall sales.
+
+* *Quantification* The digitalisation cost savings are for enterprise wide digital transformation.  A smaller but significant proportion of those savings could be allocated to digitalisation of supply chain traceability & transparency through UNTP implementation. A 1% reduction in operating costs is a conservative estimate.
+* *References*  McKinsey & Company reports on digital transformation, Deloitte insights on operational efficiency. Gartner reports on supply chain visibility, Accenture studies on inventory management. Harvard Business Review articles on data analytics, PwC reports on data-driven strategies. Forrester Research on customer experience, Bain & Company studies on customer loyalty.
```
Comment (LucaBrunello1) [inline website/docs/business-case/BusinessCaseIndustry.md:97]: All the references here are very Vague, hard to find something called " Mckinsey reports on digital transformation, etc., it's guess work.

https://www.mckinsey.com/featured-insights/mckinsey-explainers/what-is-digital-transformation 

https://newsroom.accenture.com/news/2014/accenture-supply-chain-and-inventory-management-transformation-project-improves

If these ones, you need to pay 105$ to access them:

https://store.hbr.org/product/data-analytics-collection/1064bn?sku=1064BN-BUN-ENG

Not much to be found even on PWC website:

https://www.pwc.com/gx/en/search.html?pwcGeo=gx&pwcLang=en&searchfield=data-driven+strategies

https://www.forrester.com/report/b2b-cx-for-customer-loyalty-a-guide/RES178063?ref_search=0_1731668012865

All this is extremely vague, who ever has put these references needs to put full details + link

https://www.bain.com/search/?searchValue=customer+loyalty&pageNumber=0

File: website/docs/business-case/BusinessCaseIndustry.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+* *Quantification.* The brand value benefits listed above will accrue for companies that place sustainability at the center of their corporate strategy and implement a range of measures.  UNTP implementation is only one measure but will add considerable trust to sustainability claims and can therefore conservatively account for a 1% increase in brand value.
+* *References.* Nielsen, "The Sustainability Imperative: New Insights on Consumer Expectations," 2015, Brand Finance, "Global 500 2020: The Annual Report on the World's Most Valuable Brands," 2020. RepTrak, "Global RepTrak 100: The World's Most Reputable Companies," 2019. Cone Communications, "2017 Cone Communications CSR Study," 2017.
```
Comment (LucaBrunello1) [inline website/docs/business-case/BusinessCaseIndustry.md:109]: These 2 reference could be found

https://brandfinance.com/wp-content/uploads/1/global_500_2019_locked_4.pdf 

https://www.cbd.int/doc/case-studies/inc/cs-inc-cone-communications-en.pdf

**Review by LucaBrunello1 — COMMENTED** [review]
Hi, went trough it, looks good, only comment is on the reference part of the business case for industry, some references are very vague so that it is impossible to find them or know which publication it is e.g. stuff like "Bain reports on customer loyalty", others can't be found by desk research, so perhaps who has put them has some PDF or database not accessible from the internet or something else...

**Comment by nissimsan** [comment]
@LucaBrunello1, great work! 

Pls note that this PR has already been merged. So we won't naturally come back to see and discuss your input. 

I encourage you to raise an issue referencing your input - or a PR adding your additions.

Also, do speak up if you find this process is unclear. I already have concerns if our pace and processes are understood by the community which it is intended to serve.

**Comment by LucaBrunello1** [comment]
Ok, I've created an issue https://github.com/uncefact/spec-untp/issues/221

**Pull request merged**